templates/download/_nav_secondary-v1.html

```diff
@@ -1,8 +1,9 @@
 <ul class="{% if list_class %}{{ list_class }}{% else %}nav-secondary__menu p-inline-list{% endif %}">
-  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>
-  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_2 == 'server' %} is-active"{% endif %}" href="/download/server">Server</a></li>
-  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_2 == 'desktop' %} is-active"{% endif %}" href="/download/desktop">Desktop</a></li>
-  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_2 == 'ubuntu-kylin' or level_2 == 'ubuntu-kylin-zh-CN' %} is-active"{% endif %}" href="/download/ubuntu-kylin">Ubuntu Kylin</a></li>
-  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_2 == 'alternative-downloads' %} is-active"{% endif %}" href="/download/alternative-downloads">Alternative downloads</a></li>
-  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_2 == 'ubuntu-flavours' %} is-active"{% endif %}" href="/download/ubuntu-flavours">Ubuntu flavours</a></li>
+  <li{% if not list_class %} class="p-inline-list__item"><a class="p-inline-list__link {% else %}><a class="{% endif %}{% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>
```

Comment:
@barrymcgee sorry, I misunderstood... I think this is correct...

<li{% if not list_class %} class="p-inline-list__item"><a class="p-inline-list__link {% else %}><a class="{% endif %}{% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>

if NO list_class
 <li class="p-inline-list__item"><a class="p-inline-list__link {% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>

if YES list_class
  <li><a class="{% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>

it might result in an empty class tag... but there is less logic that before